docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -190,8 +190,15 @@ data:
   ## ApplicationSet Controller Properties
   # Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.
   applicationsetcontroller.enable.leader.election: "false"
-  # "Modify how application is synced between the generator and the cluster. Default is 'sync' (create & update & delete), options: 'create-only', 'create-update' (no deletion), 'create-delete' (no update)"
-  applicationsetcontroller.policy: "sync"
+  # "Modify how application is synced between the generator and the cluster. Default is '' (empty), which means AppSets
+  # will default to the 'sync' policy (create & update & delete). Other options are:
+  # 'create-only', 'create-update' (no deletion), 'create-delete' (no update)"
```

Comment:
I know you cover it above `applicationsetcontroller.enable.policy.override` but do you think it's worthwhile to explicitly say here it will disable the ability for applicationsets to override this policy?

Reply:
Added!